assets/css/place-row.scss

```diff
@@ -5,6 +5,10 @@
   margin-bottom: 2px;
   border-bottom: 0px;
 
+  &:hover:not(.disabled) {
+    background-color: $cool-gray-40;
+  }
```

Comment:
Does applying this to `.place-row` make the hover effect apply to whole expanded rows? Do we want that?

Reply:
Good catch. Based on the designs, we don't want hover effects on expanded rows.